Vwitter/UserCell.h

```diff
@@ -0,0 +1,20 @@
+//
+//  UserCell.h
+//  Vwitter
+//
+//  Created by Christina Li on 7/11/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UserCell : UITableViewCell
```

Comment:
nit: I would try to be more consistent with using the `VW` prefix, so this should become `VWUserCell`.